### PR TITLE
Add ``become: true`` to some tasks

### PR DESCRIPTION
**Pull request by DO1JLR**
I don't like to execute ansible roles with a general ``become: yes``.
So I added ``become: true`` to some of your tasks until your role worked for me well.

Maybe you want to take it?

As far as I know it does not hurt to add the ``become: true`` option on some points. especially when your playbook needs root permissions and is usually executet as root!

### CODE REVIEW

File: tasks/bridge_configuration.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-
 - name: Check active bridge interface state
+  become: true
```
Comment (markgoddard) [inline tasks/bridge_configuration.yml]: I don't think it's necessary for this task.

File: tasks/ethernet_configuration.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-
 - name: Check active Ethernet interface state
+  become: true
```
Comment (markgoddard) [inline tasks/ethernet_configuration.yml]: I don't think it's necessary for this task.

File: tasks/bridge_configuration.yml (hunk 1 above)
Comment (DO1JLR) [reply tasks/bridge_configuration.yml]: you are right. i fixed it!

File: tasks/ethernet_configuration.yml (hunk 2 above)
Comment (DO1JLR) [reply tasks/ethernet_configuration.yml]: I fixed this too. thanks <3

**Comment by DO1JLR** [comment]
You are right. I added it now!

**Pull request merged**